e2e/libs/keywords/volume_keywords.py

```diff
@@ -79,6 +142,23 @@ def wait_for_replica_rebuilding_complete(self, volume_name, replica_node):
             replica_node
         )
 
+    def wait_for_replica_rebuilding_to_complete_on_node(self, volume_name, replica_node_type):
+        check_replica_node_type(replica_node_type)
+
+        cache_key = ResourceCache.get_replica_node_id_key_by_type(replica_node_type)
+        node_id = ResourceCache.get_link(KIND_VOLUME, volume_name, cache_key)
+        self.volume.wait_for_replica_rebuilding_complete(volume_name, node_id)
+
+    def wait_for_replica_rebuilding_to_complete(self, volume_name):
+        cached_keys = [
+            CACHE_NODE_ID_OF_REPLICA_0,
+            CACHE_NODE_ID_OF_REPLICA_1,
+            CACHE_NODE_ID_OF_REPLICA_2
+        ]
```

Comment:
As mentioned in another comment, we should not make an assumption for 3 replica only.